atom/common/node_bindings.cc

```diff
@@ -199,6 +199,22 @@ base::FilePath::StringType NodeBindings::GetHelperResourcesPath() {
   return GetResourcesPath(false).value();
 }
 
+// helper to split up NODE_OPTIONS by whitespace
+std::vector<std::string> split_string(const std::string& s,
+                                      char delimiter = ' ') {
```

Comment:
Can we use chromium's string split function here? https://cs.chromium.org/chromium/src/base/strings/string_split.h?rcl=07086b32c25cbe8af2827141a7b986c5025ab342&l=46

Reply:
Update: figured out how to use chromium's and pushed that :)